packages/features/booking-audit/lib/actions/ReassignmentAuditActionService.ts

```diff
@@ -65,8 +64,8 @@ export class ReassignmentAuditActionService
         return { isMigrated: false, latestData: validated };
     }
 
-    async getDisplayTitle({ storedData }: { storedData: { version: number; fields: z.infer<typeof fieldsSchemaV1> }; userTimeZone: string }): Promise<TranslationWithParams> {
-        const { fields } = storedData;
+    async getDisplayTitle({ storedData }: GetDisplayTitleParams): Promise<TranslationWithParams> {
+        const { fields } = this.parseStored(storedData);
```

Comment:
Curious why you made this change from passing `storedData` to method to storing it with the class.

I liked how the service was stateless so we could potentially resuse the single class for multiple audit actions. In this case multiple reassignment audit actions.